yvalenta/lib/server/server.rb

```diff
@@ -0,0 +1,46 @@
+require 'socket'
+require_relative 'connection'
+
+class Server
+
+  attr_accessor :port
+
+  def initialize(port)
+    self.port = port
+  end
+
+  def run
+    puts "Start server"
+    Socket.tcp_server_loop(port) do |sock, client_addrinfo|
```

Comment:
Unused block argument - `client_addrinfo`. If it's necessary, use `_` or `_client_addrinfo` as an argument name to indicate that it won't be used.